src/OptimisticTokenVotingPlugin.sol

```diff
@@ -62,6 +62,7 @@ contract OptimisticTokenVotingPlugin is
         bytes metadataURI;
         IDAO.Action[] actions;
         uint256 allowFailureMap;
+        // Q(danielw): I believe the code to upgrade `aggregatedL2Balance` is missing.
```

Comment:
Correct, in this version our goal was to lay all the foundation to be future compatible with L2 voting, but leaving the final L2 aggregation logic for the next stage, as we will have clarity on this.

Should this be built for the initial release?

Reply:
Nope, just making sure I understand it right.